blogs/lokiless_netobserv/index.md

```diff
@@ -0,0 +1,91 @@
+
+Below graphs shows total vCPU and memory usage for a recommended Network Observability stack (flowlogs-pipeline, eBPF-agent, Kafka and optionally Loki) for production clusters.
+
+![Compare total vCPUs utilized with and without Loki](<images/Total vCPUs consumed.png>)
+![Compare total RSS utilized with and without Loki](<images/Total Memory (RSS) consumed.png>)
+
+<sup>*</sup> actual resource utilization may depend on various factors such as flowcollector sampling size, number of workloads and nodes in an OCP cluster
+
+This comes with couple of trade-offs though, without storage of network flows it no longer provides Traffic flows table. Also, per-pod level of resource granularity is not available since it causes prometheus metrics to have high cardinality. However, should you need per-pod or per-flow level of granularity for diagnostic and troubleshooting needs, enabling loki should be pretty straightforward and both datasources can be used. When both loki and prometheus datasources are enabled, while querying on Netflow Traffic page, prometheus queried data over will be used wherever possible since it offers faster performance.
+
+## Accessing Netflow Traffic for non-admins:
+While prometheus currently doesn't supports multi-tenancy in a way that Loki does in an OpenShift cluster, non-admin users can be added to `cluster-monitoring-view`. For example, below command can be used to enable prometheus metrics visualizing for `testuser-0` user.
+
+`oc adm policy add-cluster-role-to-user cluster-monitoring-view  testuser-0`
+
+## Network Observability metrics use case:
+Let's look at a scenario how users can benefit from metrics published by Network Observability Operator. For instance, if you suspect anamaly with DNS lookups in your cluster and want to investigate workloads that may be facing DNS latencies. With Network Observability's `DNSTracking` feature and enriched prometheus metrics you can quickly set up an alert to trigger on high DNS latencies.
+
+For example, below alert will trigger for any workloads that experiences DNS latency > 100ms: 
+```yaml
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: dns-latency-alert
+spec:
+  groups:
+  - name: DNSLatencyAlert
+    rules:
+    - alert: DNSLatencyAlert
+      annotations:
+        message: |-
+          {{ $labels.DstK8S_OwnerName }} in {{ $labels.DstK8S_Namespace }} is experiencing DNS Latencies.
+        summary: "Trigger for any workloads experiencing > than 100ms DNS Latency."
+      expr: topk(7, (histogram_quantile(0.9, sum(rate(netobserv_workload_dns_latency_seconds_bucket{SrcK8S_Namespace!=""}[2m])) by (le,SrcK8S_Namespace,SrcK8S_OwnerName,DstK8S_Namespace,DstK8S_OwnerName))*1000> 100) or (histogram_quantile(0.9, sum(rate(netobserv_workload_dns_latency_seconds_bucket{DstK8S_Namespace!=""}[2m])) by (le,SrcK8S_Namespace,SrcK8S_OwnerName,DstK8S_Namespace,DstK8S_OwnerName))*1000 > 100))
+      for: 10s
```

Comment:
Not 100% sure but I think here you can keep only the second part of the query, to simplify? DNS latencies are on responses and we're displaying "{{ $labels.DstK8S_OwnerName }} in {{ $labels.DstK8S_Namespace }}" so I think we can just look at "{DstK8S_Namespace!=""}"

Also I'd remove the topk for an alert query.

```
histogram_quantile(0.9, sum(rate(netobserv_workload_dns_latency_seconds_bucket{DstK8S_Namespace!=""}[2m])) by (le,SrcK8S_Namespace,SrcK8S_OwnerName,DstK8S_Namespace,DstK8S_OwnerName))*1000 > 100
```